lib/mutant/ast/meta.rb

```diff
@@ -81,7 +44,8 @@ def assignment?
         # @api private
         #
         def attribute_assignment?
-          arguments.one? && attribute_assignment_selector?
+          !Types::METHOD_OPERATORS.include?(selector) &&
+          selector.to_s.end_with?(ATTRIBUTE_ASSIGNMENT_SELECTOR_SUFFIX)
```

Comment:
@mbj is the number of possible selectors known up-front, or is it dynamic? If it's know up-front, maybe it would be better to do `ATTRIBUTE_ASSIGNMENT_SELECTORS.include?(selector)` instead of inferring it's suitability by it's string representation?